app/controllers/hubstats/pull_requests_controller.rb

```diff
@@ -33,6 +33,7 @@ def show
       @pull_request = Hubstats::PullRequest.belonging_to_repo(@repo.id).where(id: params[:id]).first
       @comments = Hubstats::Comment.belonging_to_pull_request(params[:id]).includes(:user).created_since(@timespan).limit(20)
       @comment_count = Hubstats::Comment.belonging_to_pull_request(params[:id]).includes(:user).created_since(@timespan).count(:all)
+      @deploys = Hubstats::Deploy.where(id: @pull_request.deploy_id)
```

Comment:
We could if I wanted to make a new /tables/deploy-thingy. Currently the one I'm routing to has a length function that won't apply unless we give it some sort of array, and an association won't return an array, but instead one specific thing, so the length won't apply.